src/plugins/sync-plugin.js

```diff
@@ -252,6 +252,13 @@ const restoreRelativeSelection = (tr, relSel, binding) => {
         try {
           selection = NodeSelection.create(tr.doc, Math.min(anchor, head))
         } catch (err) {
+          // @ts-ignore
```

Comment:
Would it make sense/work to hoist this line and the one below to the top of this file?